src/components/Button/Button.styled.ts

```diff
@@ -23,7 +23,7 @@ export const ButtonWrapper = styled.span`
   }
 
   & > a {

```

Comment:
Once we update everything to be .MuiButtonBase this whole style can go away because it will depend on what variant the users decide to pass.